app/views/mailer/proposal_notification_digest.html.erb

```diff
@@ -1,59 +1,56 @@
-<td>
-  <table style="width: 100%;">
-    <tbody>
-      <tr>
-        <td style="padding-left: 10px;">
-          <h1 style="font-family: 'Open Sans','Helvetica Neue',arial,sans-serif; margin-bottom: 0;">
-            <%= t("mailers.proposal_notification_digest.title",
-                  org_name: Setting["org_name"]) %>
-          </h1>
-          <p style="font-family: 'Open Sans','Helvetica Neue',arial,sans-serif;font-size: 14px;font-weight: normal;line-height: 24px;">
-            <%= t("mailers.proposal_notification_digest.info",
-                  org_name: Setting["org_name"]) %>
-          </p>
-        </td>
-      </tr>
-    </tbody>
-  </table>
+<td style="<%= css_for_mailer_content %>">
+
+  <h1 style="<%= css_for_mailer_heading %>">
+    <%= t("mailers.proposal_notification_digest.title",
+          org_name: Setting["org_name"]) %>
+  </h1>
+
+  <p style="<%= css_for_mailer_text %>">
+    <%= t("mailers.proposal_notification_digest.info",
+          org_name: Setting["org_name"]) %>
+  </p>
 
   <% @notifications.each do |notification| %>
     <table style="width: 100%; border-top: 1px solid #DEE0E3; padding-top: 12px;">
       <tbody>
         <tr>
           <td style="padding-bottom: 20px; padding-left: 10px;">
-            <p style="font-family: 'Open Sans','Helvetica Neue',arial,sans-serif;font-size: 20px;font-weight: bold;line-height: 24px; margin: 0;">
-              <%= link_to notification.notifiable.title, proposal_url(notification.notifiable.proposal, anchor: "tab-notifications"), style: "color: #2895F1; text-decoration: none;" %>
+            <p style="<%= css_for_mailer_text %>">
```

Comment:
Note that after these changes, it's harder to know where a notification starts :thinking:.

Before these changes:

![The notification title looks like a regular link](https://user-images.githubusercontent.com/35156/171408133-5dd31735-69e7-4ad4-ad37-32512a6d2510.png)

After these changes:

![The notification title looks like a heading](https://user-images.githubusercontent.com/35156/171408171-2dd0d811-9582-4a38-b35c-1e4240a335f7.png)

I'm fine with some of the changes, like the links/buttons, and also in the original version, the text with information about the date and author was barely readable and so changing the color is great :+1:.

However, maybe we can keep the font-size of the title bigger, with the information about the date and author immediately below it?

I also wonder whether we should use an `<h2>` tag instead of a `<p>` :thinking:.